colorschemes/dark-abc.conf

```diff
@@ -0,0 +1,108 @@
+#
+# GNU General Public License
+#
+#https://github.com/geany/geany/blob/d9f8cdbad58d09f0c18ca8acccb49209263018f0/data/filedefs/filetypes.common#L2
+# Inspired by Github, Sublime Text, Geany
+
+
+
+[theme_info]
+name=Dark Abc
```

Comment:
The other themes with light/dark variants mostly use a naming scheme like "<name>-light" and "<name>-dark".

What do you think about following this scheme and rename your themes to "abc-light" and "abc-dark"?

(then also please rename the files and screenshots)

Reply:
Sorry, I didn't rename it, I uploaded it with the new name and deleted it with the old name